starport/services/network/networktypes/chainlaunch.go

```diff
@@ -0,0 +1,33 @@
+package networktypes
+
+import launchtypes "github.com/tendermint/spn/x/launch/types"
+
+// ChainLaunch represents the launch of a chain on SPN
+type ChainLaunch struct {
+	ID          uint64
+	ChainID     string
+	SourceURL   string
+	SourceHash  string
+	GenesisURL  string
+	GenesisHash string
+	CampaignID  uint64
+}
+
+// ParseChainLaunch parses a chain launch data from SPN and returns a ChainLaunch object
+func ParseChainLaunch(chain launchtypes.Chain) ChainLaunch {
```

Comment:
```suggestion
func ToChainLaunch(chain launchtypes.Chain) ChainLaunch {
```

Reply:
I have a personal preference for `Parse` since it represents an action

Maybe @Pantani @ivanovpetr @fadeev opinion can be worth since it may influence the naming for future similar methods

Reply:
parse means parsing a format into a structure whereas here we just convert types

Reply:
We have the same wording issue in the `prepare` PR for the genesis account type, we can decide it on this one

There is still some parsing to me since we convert the type into a simpler format, like reading the initial genesis into a string `genesisURL` for example

`To<Name>` doesn't reflect an action to me

> I like more verbose, as ChainToChainLaunch

The type is `Chain` in `spn` but for genesis account it would become for example `GenesisAccountToGenesisAccount`